integration-tests/src/test/java/oracle/weblogic/kubernetes/ItFmwMiiDomain.java

```diff
@@ -213,6 +227,25 @@ public void testFmwModelInImage() {
         fmwMiiImage);
 
     createDomainAndVerify(domain, fmwDomainNamespace);
+
+    //verify the introspector pod is created and runs
+    logger.info("Verifying introspector pod is created, runs");
+    String introspectPodNameBase = getIntrospectJobName(domainUid);
+    logger.info("Checking introspector pod exists and introspect version");
+    checkPodExists(introspectPodNameBase, domainUid, fmwDomainNamespace);
+    String introspectVersion1 =
+        assertDoesNotThrow(() -> getCurrentIntrospectVersion(domainUid, fmwDomainNamespace));
+    logger.info("Before restarting operator introspectVersion is: " + introspectVersion1);
+
+    logger.info("Is going to restart operator in the namespace: " + opNamespace);
+    restartOperator(opNamespace);
+    //verify introspectorVersion does not change
+    checkPodExists(introspectPodNameBase, domainUid, fmwDomainNamespace);
```

Comment:
what is the guarantee that the introspector pod will still be running? It might have finished running before operator is restarted.

Reply:
Here we pickup FMW domain because we know FMW domain has long running introspector job, usually taking around 5 mins and we have introspectorJobActiveDeadlineSeconds(600L).
So under normal circumstance introspector pod would still be running after operator is restarted. And after the operator is restarted I verify the intropsector pod is still running.
But you are right the testing logic itself doesn't guarantee that.
Maybe I should fail the test if after operator restarting the introspector pod doesn't exist?